### ISSUE TITLE
Zoom MacOS: `win_open` event doesn't fire until new meeting window is blurred then re-focused

### ISSUE THREAD

**Issue by pokey** [talonvoice/talon#606]
When I launch a Zoom meeting, the `win_open` event doesn't fire until I blur the window and then re-focus it.  Here is the log from a `ui.register("", lambda *args, **kwargs: print(args, kwargs))`:

```
2023-04-15 18:20:25    IO event ('element_focus', <AXWindow 'Zoom Meeting'>) {}
2023-04-15 18:20:27    IO event ('app_launch', App(pid=7044, "CptHost")) {}
2023-04-15 18:20:32    IO event ('app_deactivate', App(pid=4785, "zoom.us")) {}
2023-04-15 18:20:32    IO event ('app_activate', App(pid=84779, "Talon")) {}
2023-04-15 18:20:32    IO event ('win_focus', Window(app=App(pid=84779, "Talon"), title="Talon Log Viewer")) {}
2023-04-15 18:20:38    IO event ('app_deactivate', App(pid=84779, "Talon")) {}
2023-04-15 18:20:38    IO event ('app_activate', App(pid=4785, "zoom.us")) {}
2023-04-15 18:20:38    IO event ('win_focus', Window(app=App(pid=4785, "zoom.us"), title="Zoom")) {}
2023-04-15 18:20:39    IO event ('win_open', Window(app=App(pid=4785, "zoom.us"), title="Zoom Meeting")) {}
2023-04-15 18:20:39    IO Window(app=App(pid=4785, "zoom.us"), title="Zoom Meeting") App(pid=4785, "zoom.us")
2023-04-15 18:20:39    IO event ('win_focus', Window(app=App(pid=4785, "zoom.us"), title="Zoom Meeting")) {}
2023-04-15 18:20:40    IO event ('app_deactivate', App(pid=4785, "zoom.us")) {}
2023-04-15 18:20:40    IO event ('app_activate', App(pid=84779, "Talon")) {}
2023-04-15 18:20:40    IO event ('win_focus', Window(app=App(pid=84779, "Talon"), title="Talon Log Viewer")) {}
```

I launched the meeting at 18:20:25, on that first `element_focus`.  Note how there is no `win_open`.  Then ~10 seconds later, I focus the log viewer and back to the meeting and get the `win_open` that I would have expected to receive in the first place.

Version: 0.3.1-551-g3de1, Branch: beta, OS: macOS-13.2-arm64-arm-64bit

**Comment by nriley** [comment]
This seems to be a special case of the more general "Talon doesn't notice when a Mac app opens a window" that I've had to work around extensively (often by polling and changing focus to try to get Talon to "notice" the window). The Zoom meeting window also doesn't show up in app.windows() until you activate the app. For another example, see #315.

**Comment by nriley** [comment]
This is fixed as of the latest beta (see #502 and #315).
